aspnetcore/fundamentals/configuration/platform-specific-configuration/sample/StartupDiagnostics/StartupDiagnostics.csproj

```diff
@@ -1,44 +1,44 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.1.0" />
   </ItemGroup>
 
   <Target Name="CopyTarget" AfterTargets="Build">
     <Message Importance="High" Text="Creating directory for assembly if it doesn't exist ..." />
-    <MakeDir Directories="$(USERPROFILE)\.dotnet\store\x64\netcoreapp2.0\StartupDiagnostics\1.0.0\lib\netcoreapp2.0\" 
-             Condition="!Exists('$(USERPROFILE)\.dotnet\store\x64\netcoreapp2.0\StartupDiagnostics\1.0.0\lib\netcoreapp2.0\')" />
+    <MakeDir Directories="$(USERPROFILE)\.dotnet\store\x64\netcoreapp2.1\StartupDiagnostics\1.0.0\lib\netcoreapp2.1\" 
```

Comment:
Consider replacing occurrences of "netcoreapp2.1" with `$(TargetFramework)` to improve maintainability. When the TFM changes above, you don't have to update all these locations too.

Reply:
@scottaddie Great idea. How about we continue with ...

```xml
<SharedFrameworkVersion>2.1.0</SharedFrameworkVersion>
<StartupDiagnosticsVersion>1.0.0</StartupDiagnosticsVersion>
```
... and ... `$(SharedFrameworkVersion)` + `$(StartupDiagnosticsVersion)`